conf/outputs.json

```diff
@@ -2,6 +2,9 @@
   "aws-lambda": {
     "sample-lambda": "function-name:qualifier"
   },
+  "aws-lambda-v2": [
```

Comment:
IIRC, this is not actually how this info will be written to the `outputs.json` config file for `AWSOutput` types .. see here:

https://github.com/airbnb/streamalert/blob/41da6b5514b4949c04cab90aa85631fa833634d0/streamalert/alert_processor/outputs/aws.py#L50-L69

did you confirm that this actually works as expected with the `manage.py outputs new ... `command?

Reply:
Oh wow, no this is totally not gonna work. Good catch. No I didn't confirm it as I'm still testing it, but it's good to know that it's definitely wrong. 

I think the easy fix is to just not inherit from this base class and just use the `OutputDispatcher` base class like any other sane output.

Reply:
agreed - we can trend away from the AWSOutput class usage for future aws outputs now that ssm is a thing